python/mooseutils/unit_tests/tests

```diff
@@ -0,0 +1,32 @@
+[Tests]
```

Comment:
Yes, to get this working we would need a `run_tests` in `moose/python` as well as the `RunPythonApp` tester. I couldn't find a way to get `run_tests` to run in some other directory. It sets its working directory to where the script lives.
I wasn't sure if you wanted all that to be mixed in with this PR.